### PR TITLE
Add cephadm_commands placeholder variable

### PR DESCRIPTION
**Pull request by jovial**
Support for this feature was added in #290. This adds a variable and some documentation to raise awareness of its existence.

### CODE REVIEW

File: doc/source/configuration/cephadm.rst [hunk 1]
```diff
@@ -239,6 +239,21 @@ for Cinder, Cinder backup, Glance, and Nova in Kolla Ansible.
          mgr: "profile rbd pool=images"
        state: present
 
+Ceph Commands
+~~~~~~~~~~~~~
+
+It is possible to run an arbitrary list of commands against the cluster after deployment
+by setting the ``ceph_commands`` variable. ``ceph_commands`` should be a list of commands
```
Comment (markgoddard) [inline doc/source/configuration/cephadm.rst]:
```suggestion
by setting the ``cephadm_commands`` variable. ``cephadm_commands`` should be a list of commands
```

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: doc/source/configuration/cephadm.rst [hunk 2]
```diff
@@ -239,6 +239,21 @@ for Cinder, Cinder backup, Glance, and Nova in Kolla Ansible.
...
+to pass to ``cephadm shell -- ceph``. For example:
+
+.. code:: yaml
+
+   # A list of commands to pass to cephadm shell -- ceph. See stackhpc.cephadm.commands
+   # for format.
+   cephadm_commands:
+     - "fs new cephfs cephfs_metadata cephfs_data"
```
Comment (mnasiadka) [inline doc/source/configuration/cephadm.rst]: I would argue that's a good example, changing placement should rather be done using cephadm role, not the commands role.

Comment (jovial) [reply doc/source/configuration/cephadm.rst]: That is the example from the docs in the collection. I was going to use this approach to do:

```
ceph config set mgr mgr/prometheus/server_addr 10.0.0.1
```

So it wasn't bound to all interfaces. Is that a better example?

EDIT: Assuming you meant **not** a good example

Comment (mnasiadka) [reply doc/source/configuration/cephadm.rst]: Yes, an mgr var is a good example :-) I meant the old one is NOT a good example.

Comment (jovial) [reply doc/source/configuration/cephadm.rst]: Thanks, makes sense to not encourage a bad pattern of usage.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**